project.mml

```diff
@@ -1529,6 +1529,7 @@ Layer:
           OR tags->'compressed_air'='yes'
           OR tags->'car_wash'='yes'
           OR (amenity='motorcycle_parking' AND (bicycle='yes' OR bicycle='designated'))
+          OR (amenity='charging_station' AND bicycle='yes')
```

Comment:
`bicycle='designated'` as well (future-proof, not very important at the moment apparently)?

Reply:
oui, j'y ai pensé, mais il y en a quatre dans le monde, donc bof